tests/cli/test_login_list.py

```diff
@@ -58,6 +58,7 @@ def test_data_passing(monkeypatch):
         ]
         prompter.assert_not_called()
 
+    @staticmethod
     def test_no_remote_configs(monkeypatch, capsys):
```

Comment:
The only way this worked so far is `monkeypatch` is unused
```suggestion
    @staticmethod
    def test_no_remote_configs(capsys):
```